CONTRIBUTING.md

```diff
@@ -11,10 +11,16 @@ First of all, thank you so much for taking your time to contribute! We always we
 You can install Promgen for a development environment as follows.
 
 ```bash
-virtualenv --python=/path/to/python3 /path/to/virtualenv
-source /path/to/virtualenv/activate
+python3 -m venv /path/to/virtualenv
+source /path/to/virtualenv/bin/activate
 pip install -e .[dev]
 pip install mysqlclient # psycopg or another database driver
+# Provide initial configuration
+promgen bootstrap
+# Enable DEBUG (and development) mode
+echo 1 > ~/.config/promgen/DEBUG
+# Copy default settings (amend by hand after)
+cp promgen/tests/examples/promgen.yml
```

Comment:
This should be copied by bootstrap already, though perhaps I could have it automatically open up in $EDITOR if you thought that would be useful

Reply:
Thanks for the feedback! You're right, bootstrap will already have written a config file.
Do you know if there is a standard command on UNIX systems to start the default editor? I know debian provides `editor` which maps to nano, vim, emacs, ... but I'm not sure about other distributions.

Reply:
EDITOR is the one I have seen that is most common though sometimes I have seen VISUAL. Trying to find a more definitive source but it can be difficult with things like this :)

Reply:
Alright, perhaps a comment is feasible instead of dissecting UNIX standard practices ;)
On my system `$EDITOR` is unset when starting the shell and the `editor` executable references nano.

Reply:
Yeah, it's just a random thought I had. Feel free to keep things simple :)